doc/api/esm.md

```diff
@@ -654,7 +654,7 @@ CommonJS and ES module instances of the package:
 
     ```js
     // ./node_modules/pkg/index.mjs
-    export state from './state.cjs';
+    export { default as state } from './state.cjs';
```

Comment:
Would it maybe make sense to be a bit more verbose here to match what is going on above?
```suggestion
import state from './state.cjs';
export {
  state
};
```